integration/ooo_ingestion_test.go

```diff
@@ -168,6 +168,12 @@ func TestOOOHistogramIngestionDisabled(t *testing.T) {
 		map[string]string{
 			"-ingester.out-of-order-time-window":            "10m",
 			"-ingester.native-histograms-ingestion-enabled": "true",
+			// Default block-ranges-period for integration tests is 1m
```

Comment:
nit: let's add `"-ingester.ooo-native-histograms-ingestion-enabled": "false"`  while we're at it. Just in case we change the default/remove the flag, make the test fail more explicitly.

Reply:
Updated in https://github.com/grafana/mimir/pull/9592/commits/aeb921c607493208bd9ae8d68f267dc448584c1e